### PR TITLE
Update Malayalam translation

### PR DESCRIPTION
**Pull request by rathodroshan0137**
notepad-plus-plus/PowerEditor/installer/nativeLang/Malayalam.xml

### CODE REVIEW

**Comment by rathodroshan0137** [comment]
@SinghRajenM  Hello Sir, Could you please let me know Update Malayalam translation in notepad plus?

**Comment by rathodroshan0137** [comment]
@donho Hello Sir,
Could you please check this new translation for notepad-plus-plus?

**Comment by rathodroshan0137** [comment]
@SinghRajenM Could you please assign this to don ho?

**Comment by rathodroshan0137** [comment]
@donho please have a look Malayalam translation?

**Comment by SinghRajenM** [comment]
@rathodroshan0137 did you really test your changes??? Below are the thumb rules - 
1. **PR must be tested well.**
2. It should be single commit. If you're updating because of some reason, make sure you use "amend" and "force" flag so that it will be appended to previous commit.
3. Submit PR from a branch instead of master branch.

image

image

image

image

image

File: PowerEditor/installer/nativeLang/Malayalam.xml [hunk 1]
```diff
@@ -0,0 +1,1176 @@
...
+അവ തുറക്കാൻ നിങ്ങൾക്ക് ഉറപ്പാണോ?"/>
+            <SettingsOnCloudError title="ക്ലൗഡിൽ ക്രമീകരണം" message="വായന മാത്രമുള്ള ഡ്രൈവ് അടിസ്ഥാനത്തിൽ ക്ലൗഡിൽ സജ്ജീകരണ ക്രമീകരണങ്ങൾ ഏർപ്പെടുമെന്ന് തോന്നുന്നു,
+അല്ലെങ്കിൽ ഫോൾഡറിൽ ആവശ്യമായ ആക്സസ് എഴുതാൻ അവകാശം വേണം .
+ക്ലൗഡിൽ നിങ്ങൾക്കുള്ള ക്രമീകരണങ്ങൾ റദ്ദാക്കപ്പെടും. മുൻഗണന എന്ന ഡയലോഗിലൂടെ ഒരു സഹജമായ മൂല്യം ദയവായി പുനഃസജ്ജീകരിക്കുക ."/>
+            <FilePathNotFoundWarning title="ഫയൽ തുറക്കുക" message="നിങ്ങൾ തുറക്കാൻ ശ്രമിക്കുന്ന ഫയൽ നിലവിലില്ല ."/>
+            <SessionFileInvalidError title="സെഷൻ ലോഡുചെയ്യാൻ കഴിഞ്ഞില്ല" message="ഒന്നുകിൽ സെഷൻ ഫയൽ കേടായി അല്ലെങ്കിൽ സാധുതയുള്ളതല്ല ."/>
+            <DroppingFolderAsProjectModeWarning title="അസാധുവായ പ്രവർത്തനം" message="നിങ്ങൾക്ക് ഫോൾഡർ പ്രോജക്ട് മോഡ് ആയി ഉപേക്ഷിക്കുന്നതിനാൽ, നിങ്ങൾക്ക് ഫയലുകളും ഫോൾഡറുകളും മാത്രമേ ഡ്രോപ്പ് ചെയ്യാൻ കഴിയൂ .
+നിങ്ങൾ & quot; ഫോൾഡർ ഡ്രോയിംഗ് & quot; വർക്ക് സ്പെയ്സ് ആയി ഫോൾഡർ സമാരംഭിക്കുന്നതിനുപകരം & quot; ഫോൾഡറിന്റെ എല്ലാ ഫയലുകളും തുറക്കുക & quot; & quot; സ്ഥിരസ്ഥിതി ഡയറക്ടറി & quot; ഈ ഓപ്പറേഷൻ പ്രവർത്തിക്കുന്നതിന് മുൻഗണനകളുടെ ഡയലോഗിന്റെ വിഭാഗം ."/>
```
Comment (SinghRajenM) [inline PowerEditor/installer/nativeLang/Malayalam.xml]: > & quot; ഫോൾഡർ ഡ്രോയിംഗ് & quot;

There should be no space between '&' and quot;

File: PowerEditor/installer/nativeLang/Malayalam.xml [hunk 2]
```diff
@@ -0,0 +1,1176 @@
...
+            <SortingError title="സോർട്ടിംഗ് പിശക്" message="$ INT_REPLACE $ എന്ന വരിമൂലമുള്ള സാംഖിക തരം ക്രമീകരിക്കാൻ കഴിഞ്ഞില്ല ."/>
+            <ColumnModeTip title="നിര മോഡ് ടിപ്പ്" message="ദയവായി & quot; ALT + മൗസ് തിരഞ്ഞെടുക്കൽ & quot; ഉപയോഗിക്കുക അല്ലെങ്കിൽ & quot; Alt + Shift + അമ്പടയാളം കീ & quot; കോളം മോഡിലേക്ക് മാറുന്നതിന് ."/>
+            <BufferInvalidWarning title="സംരക്ഷിക്കുന്നത് പരാജയപ്പെട്ടു" message="സംരക്ഷിക്കാനാവില്ല: ബുഫർ അസാധുവാണ് ."/>
+            <DoSaveOrNot title="രക്ഷിക്കും" message="ഫയൽ & quot; $ STR_REPLACE $ & quot; ?"/>
```
Comment (SinghRajenM) [inline PowerEditor/installer/nativeLang/Malayalam.xml]: > & quot; $ STR_REPLACE $ & quot;

all the spaces should be removed. similarly other places as well.

**Comment by donho** [comment]
No translation is better than a bad translation.

**Pull request closed without merging**